### DISCUSSION TITLE
Status of support for inherited members

### DISCUSSION THREAD

**Discussion by jayqi** [mkdocstrings/mkdocstrings#536] [General]
In the [migration notes for 0.19](https://mkdocstrings.github.io/handlers/overview/#selection-options), I see the note:

> `inherited_members` is not yet implemented.

As far as I can tell, neither the new experimental `python` handler, nor the old `python-legacy` support this option right now. Am I missing something?

If it's the case that `inherited_members` is a casualty of handler changes, is there a timeline for when to expect inherited members? As someone using mkdocstrings before the recent handler changes and configuration interface changes, it feels like a significant regression to not have this functionality at all. 

This is important functionality that I need for my documentation, as I have abstract base classes that implement the bulk of the API interface, but the concrete classes are what users actually use and will look up documentation for. 

Thanks!

**Comment by pawamoy** [comment]
Hello! No there's no timeline. I'm currently getting back to open source after a (subjectively) long burnout. I'm not in capacity to build up a timeline yet.

I agree that supporting inherited members is an important feature. It's definitely on the "to do" list.

**Comment by pawamoy** [comment]
Now supported :slightly_smiling_face: https://mkdocstrings.github.io/python/usage/configuration/members/#inherited_members
